src/public/services/FieldVerificationService.ts

```diff
@@ -0,0 +1,100 @@
+import axios from 'axios'
+import { Opaque } from 'type-fest'
+
+export type JsonDate = Opaque<string, 'JsonDate'>
```

Comment:
is this level of differentiation really necessary, given that we always pass `expireAt` in an object? I can see the purpose if e.g. there's a function which takes in multiple args of the same type, and it's very important not to mix them up.

conversely, if we just use a `string`, this module can import the `Transaction` type from `src/types`, which iirc was the purpose of creating the `types` directory outside of `app`.

Reply:
Good practice to start, so we don't just pass any random string into functions that expect a date-string. This entire type is also stringified (due to backend->frontend serialization), and thus the `Transaction` type cannot/shouldn't be used